mmhuman3d/models/utils/SMPLX.py

```diff
@@ -0,0 +1,514 @@
+from typing import List
+from mmhuman3d.core.conventions.keypoints_mapping import get_keypoint_idx, get_keypoint_idxs_by_part
+from mmhuman3d.utils.geometry import weak_perspective_projection
+from mmhuman3d.utils.transforms import rotmat_to_rot6d
+from smplx.utils import find_joint_kin_chain
+import torch
+import torch.nn.functional as F
+
+
+class SmplxHandMergeFunc():
```

Comment:
It will be good to describ what this function does.

Reply:
Descriptions added.